src/site/content/de/de.11tydata.js

```diff
@@ -0,0 +1,31 @@
+const livePaths = require('../../_filters/live-paths');
+const allPaths = require('../../_data/paths');
+const lang = require('./lang');
+
+// =============================================================================
+// HOME OVERVIEW
+//
+// This is the context object for the homepage.
+// It helps layout cards featured on the homepage, and their ordering.
+//
+// =============================================================================
+
+module.exports = function () {
+  const paths = [
+    allPaths['vitals'],
+    allPaths['progressive-web-apps'],
+    allPaths['accessible'],
+    allPaths['fast'],
+    allPaths['reliable'],
+    allPaths['secure'],
+  ].filter(livePaths);
+
+  return {
+    lang: lang.lang,
+    locale: lang.locale,
+    // translated: 'none',
```

Comment:
This is needed for the UI to know that it's not an original post but a translation.